CHANGELOG.md

```diff
@@ -27,10 +27,21 @@ for i18next, solely on the front end for admin UI purposes.
 * The syntax of the method defined for dynamic `choices` now accepts a module prefix to get the method from, and the `()` suffix.  
 This has been done for consistency with the external conditions syntax shipped in the previous release. See the documentation for more information.
 * Do not display "required" errors for hidden fields.
+* Add `viewPermission`/`editPermission` property on schema field. You can now decide if a schema field requires
```

Comment:
Added the `viewPermission` property of schema fields, and renamed `permission` to `editPermission` (with backwards compatibility) for clarity